src/main/java/net/coderbot/iris/postprocess/CompositeRenderer.java

```diff
@@ -234,8 +234,7 @@ private Program createProgram(ProgramSource source, ImmutableSet<Integer> flippe
 		}
 
 		// TODO: Don't duplicate this with FinalPassRenderer
-		// TODO: Parse the value of const float centerDepthSmoothHalflife from the shaderpack's fragment shader configuration
-		builder.uniform1f(UniformUpdateFrequency.PER_FRAME, "centerDepthSmooth", this.centerDepthSampler::getCenterDepthSmoothSample);
+		builder.addDynamicSampler(centerDepthSampler::getCenterDepthTexture, "iris_centerDepthSmooth");
```

Comment:
I notice that in this version we don't let gbuffer or shadow programs read the center depth value, as in older versions of Iris. I assume that ShadersMod / OptiFine let shaders do this, likely just returning the value from the previous frame. Do we want to look into that at some point? Doesn't have to be in this PR.